app/components/ransack/hidden_field_component.html.erb

```diff
@@ -0,0 +1 @@
+<input type="hidden" name="q[s]" id="q_s" value='<%="#{ransack_obj.sorts[0].name} #{ransack_obj.sorts[0].dir}"%>'>
```

Comment:
**nitpick (blocking):** Can you change this to use `fields_for` and `hidden_field`, see below:

```
<%= fields_for ransack_obj do |ransack_fields| %>
  <%= ransack_fields.hidden_field :s,
                              value:
                                "#{ransack_obj.sorts[0].name} #{ransack_obj.sorts[0].dir}" %>
<% end %>
```

This means that if we use a different name then `q` this will still work.

Reply:
Thanks for this, I was trying to figure out something like this but with no luck. Fixed in 48af86ae30b3a0a4640bf9c11b95e6d3a6c46b7b